mapadroid/webhook/webhookworker.py

```diff
@@ -479,6 +481,23 @@ def __prepare_mon_data(self, mon_data):
                 if self.__args.quest_webhook_flavor == "poracle":
                     mon_payload["weather"] = mon["weather_boosted_condition"]
 
+            if mon["seen_type"] in ("nearby_stop", "lure_wild", "lure_encounter"):
```

Comment:
Enum for the "seen_type" would make sense imho

Reply:
The way I'd do it:
1. Define the enum in `utils/madGlobals.py`
2. Convert selected values to enum in the DbWebhookReder.get_mon_changed_since method
3. Introduce a new key to the data returned by that method, which is then used in those if-statements

that's good?

Reply:
Would work for me I guess. Would probably just work with enum's .value access for now as well